src/utils/socket.js

```diff
@@ -0,0 +1,18 @@
+import { io } from '../index';
+
+/**
+ * Class representing the helper methods
+ * @class HelperMethods
+ * @description methods used everywhere in the codebase
+ */
+export default class Sockets {
+  /**
+  * A method used to send server errors
+  * @param {object} event - HTTP response object
+  * @param {String} message - The error message you want to set.
+  * @returns {object} res - The HTTP response object
+  */
+  static emiter(event, message) {
+    return io.emit(event, message);
```

Comment:
Do we need a class to call just io.emit ? I think importing io to the request controller would do the job

Reply:
yes....i just did so that i can be reused

Reply:
Okay